rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -541,6 +541,11 @@ def element_attribute(element, name)
           execute :getElementAttribute, id: element.ref, name: name
         end
 
+        # Backwards compatibility for w3c
+        def element_property(element, name)
+          execute :getElementAttribute, id: element, name: name
```

Comment:
This is actually the fallback, the getElementProperty is already implemented [here](https://github.com/SeleniumHQ/selenium/blob/master/rb/lib/selenium/webdriver/remote/w3c_bridge.rb#L528) but there was no path to calling it. Should this execute js in order to get the property value like you implemented it in python instead?